crowdin_api/api_resources/distributions/resource.py

```diff
@@ -21,7 +21,6 @@ def get_distributions_path(self, projectId: int, hash: Optional[str] = None):
 
     def list_distributions(
         self,
-        projectId: int,
```

Comment:
This is breaking change. The `projectId` parameter should be optional in all APIs.